server/historian/package.json

```diff
@@ -44,7 +44,7 @@
     "temp-directory": "nyc/.nyc_output"
   },
   "devDependencies": {
-    "@fluidframework/build-tools": "^0.2.25384",
```

Comment:
does the lock file needs to be updated here as well?

Reply:
Huh.  Never realized 'Historian' isn't included in 'fluid-build --all --install'.  Thanks for pointing that out.

Looks like the Historian 'package-lock.json' is pretty stale.  Maybe I'll do that one in a follow up commit so it's easy to pinpoint if the minor version bumps cause trouble.

Reply:
Followup here: #6767